pytripgui/plan_vc/plan_cont.py

```diff
@@ -0,0 +1,183 @@
+
+        model.kernel = view.get_selected_krenel()
+        model.target_dose = view.get_target_dose_value()
+        model.target_dose_percent = view.get_relative_target_dose_value()
+
+        model.iterations = view.get_iterations_value()
+        model.eps = view.get_eps_value()
+        model.geps = view.get_geps_value()
+        model.opt_method = view.get_selected_opti_method()
+        model.opt_principle = view.get_selected_principle()
+        model.dose_alg = view.get_selected_dose_algorithm()
+        model.bio_alg = view.get_selected_bio_algorithm()
+        model.opt_alg = view.get_selected_opti_algorithm()
+
+        model.want_phys_dose = view.get_physical_dose_dist_state()
+        model.want_bio_dose = view.get_biological_dose_dist_state()
+        model.want_dlet = view.get_dose_averaged_let_state()
+        model.want_rst = view.get_raster_scan_file_state()
+
+    def _setup_target_roi(self):
+        self._fill_view_with_rois()
+        self._set_correct_target_roi_view()
+
+    def _fill_view_with_rois(self):
+        view = self.view
+
+        for voi in self.vois:
+            view.add_target_roi_with_name(voi, voi.name)
+
+    def _set_correct_target_roi_view(self):
+        model = self.model
+        view = self.view
+
+        if model.voi_target is not None:
+            view.set_target_roi_to_this(model.voi_target)
+
+    def _setup_oar(self):
+        self._fill_view_with_oars()
+        self._mark_specified_oars_as_checked()
+
+    def _fill_view_with_oars(self):
+        view = self.view
+
+        for voi in self.vois:
+            view.add_oar_with_name(voi, voi.name)
+
+    def _mark_specified_oars_as_checked(self):
+        for oar in self.model.vois_oar:
+            self.view.set_oar_as_checked(oar)
+
+    def _setup_target_tissue(self):
+        pass        # TODO
+
+    def _setup_residual_tissue(self):
+        pass        # TODO
+
+    def _setup_kernels(self):
+        view = self.view
+        model = self.model
+        kernels = self.kernels
+
+        for kernel in kernels:
+            view.add_kernel_with_name(kernel, kernel.name)
+
+        view.select_kernel_view_to_this(model.kernel)
+
+    def _setup_optimization_metod(self):
+        view = self.view
+        opt_methods = self.model.opt_methods
+
+        for i, key in enumerate(opt_methods):
+            method_name = opt_methods[key][1]
+            view.add_opti_method_with_name(key, method_name)
+
+        view.select_opti_method_view_to_this(self.model.opt_method)
+
+    def _setup_principle(self):
+        view = self.view
+        principles = self.model.opt_principles
+
+        for i, key in enumerate(principles):
+            principle_name = principles[key][1]
+            view.add_principle_with_name(key, principle_name)
+
+        view.select_principle_view_to_this(self.model.opt_principle)
+
+    def _setup_dose_algorithm(self):
+        view = self.view
+        dose_alg = self.model.dose_algs
```

Comment:
if it is a list (or other iterable), I prefer to have a plural "s" in the variable name. Makes code more readbale `for dose_alg in dose_algs` etc.

Reply:
I agree !

Reply:
ok